documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,53 @@
+== External Trigger
+=== IMX296
+The global shutter camera can be triggered externally by pulsing the XTR pin on the board.
+Multiple cameras can be connected to the same pulse leading to a better way to synchronise two cameras.
+
+The low puslewidth is equal to the exposure time. i.e. a low pulse of 10ms leads to an exposure time of 10ms.
+
+==== Preparation
+
+First, remove/desolder resistor 22 on the GS Camera board. This connects GP0 to XTR and without removal, the camera will not operate
+in external trigger mode.
+
+Next solder a wire each to the touchpoints of XTR and GND on the GS Camera board
+Connect these to the pico - GND to GND, XTR to any pin. Pin 28 is used in this example.
+==== Boot up the Raspberry Pi with the camera connected.
+
+Enable external triggering through superuser mode:
+[,bash]
+----
+sudo su
+echo 1 > /sys/module/imx296/parameters/trigger_mode
+exit
+----
+
+==== Raspberry Pi Pico Code
+```
+from machine import Pin, PWM
+
+from time import sleep
+
+pwm = PWM(Pin(28))
+
+framerate = 30
+shutter = 6000  # In microseconds
+
+frame_length = 1000000 / framerate
+pwm.freq(framerate)
+
+pwm.duty_u16(int((1 - shutter / frame_length) * 65535))
+```
+The low pulsewidth is equal to the shutter time, and the frequency of the PWM equals the framerate.
+
+==== Operation
+
+Run the code on the pico, and run the code for the camera:
+[,bash]
+----
+libcamera-hello -t 0 --qt-preview --shutter 3000
+----
+
+==== Note
+When running libcamera apps, you will need to specify a fixed shutter duration, so that the AGC controller
+knows not to change the shutter speed during exposure control. This will force the camera to only change gain.
```

Comment:
> This will force the camera to only change gain.

I would replace with something like:

This would ensure the AGC does not try adjusting camera's shutter speed, which is controlled by the external trigger pulse.